### ISSUE TITLE
No License

### ISSUE THREAD

**Issue by tejasvikothapalli** [stemkoski/stemkoski.github.com#41]
This is awesome!

Please put license so people can use the awesome features for their applications legally!

**Comment by RoboPython** [comment]
Yeah agreed, I was going to email you about this but I see someone has raised it here

**Comment by ashl1** [comment]
This is duplicate of #18
